services/ui-src/src/components/fields/DropdownField.tsx

```diff
@@ -35,26 +36,31 @@ export const DropdownField = ({
   styleAsOptional,
   ...props
 }: Props) => {
-  const { report, updateReport } = useContext(ReportContext);
+  const { report, updateReport, submittedReportsByState } =
+    useContext(ReportContext);
   const { entities, entityType, updateEntities, selectedEntity } =
     useContext(EntityContext);
   const { full_name, state } = useUser().user ?? {};
 
   // fetch the option values and format them if necessary
   const formatOptions = (options: DropdownOptions[] | string) => {
     let dropdownOptions = [];
-    if (typeof options === "string") {
-      const dynamicOptionValues = report?.fieldData[options];
-      if (dynamicOptionValues) {
-        const fieldOptions = dynamicOptionValues.map((option: EntityShape) => ({
+    if (options === "copyEligibleReports") {
+      dropdownOptions =
+        submittedReportsByState?.map((option) => ({
+          label: `${option.programName} ${convertDateUtcToEt(option.dueDate)}`,
+          value: option.fieldDataId!,
+        })) ?? [];
```

Comment:
If this chunk is the only place we're using submittedReportsByState, would it make more sense to just filter for submitted reports here, rather than every time the reports array gets rev'd?

Reply:
I think it makes more sense in the ReportContext because I don't want a field component to own something like that. May end up using it in other spots in the future too. Also not a very costly operation to filter them down in the state each time IMO.